Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM jupyter/base-notebook:latest
+
+LABEL maintainer="Fabian Brandt-Tumescheit <brandtfa@hu-berlin.de>"
+
+USER root
+RUN mkdir /var/lib/apt/lists/partial && \
+    apt-get update && \
+    apt-get install -y  --no-install-recommends cmake git build-essential && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+USER 1000
+RUN pip install --upgrade pip setuptools cython
+RUN pip install powerlaw
+RUN pip install sklearn
+RUN pip install seaborn
+RUN pip install networkit
+RUN pip install pandas
+RUN pip install tabulate
+RUN pip install matplotlib
+RUN pip install networkx
+RUN pip install ipycytoscape
+
+RUN git clone https://github.com/networkit/networkit.git
+WORKDIR networkit
+RUN git checkout 8.0
+RUN mv notebooks ${HOME}/
+RUN mv input ${HOME}/
+RUN rm ${HOME}/notebooks/test_notebooks.py
```

Comment:
Hm?

Reply:
The idea is that the image-size is minimal. Like written above - the repository where the Dockerfile is hosted, is not part of the final image. The intuition is: check out our repo in the building process, but only include files you want to "use" in the end. So the user is only interested in input-graphs and notebooks. The rest is removed afterwards.